pkg/operator/staticpod/controller/revision/revision_controller.go

```diff
@@ -191,18 +198,20 @@ func (c RevisionController) isLatestRevisionCurrent(revision int32) (bool, strin
 			existingData = existing.Data
 		}
 		if !equality.Semantic.DeepEqual(existingData, requiredData) {
+			resourceDiffString := resourceapply.JSONPatch(existing, required)
 			if klog.V(4) {
-				klog.Infof("secret %q changes for revision %d: %s", s.Name, revision, resourceapply.JSONPatch(existing, required))
+				klog.Infof("secret %q changes for revision %d: %s", s.Name, revision, resourceDiffString)
```

Comment:
do we leak security critical information?

Reply:
yeah, was thinking the same... I think we might even now on loglevel 4 ;-)